MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -218,6 +221,36 @@ extension UIApplicationState {
     }
 }
 
+extension AVAudioSession {
+    var audioType: String {
+        if bluetoothAudioConnected() {
+            return "bluetooth"
+        }
+        if headphonesConnected() {
+            return "headphones"
+        }
+        return "speaker"
```

Comment:
Is there a specific check we can do for `speaker` and fallback to `other` if nothing else is found?